### PR TITLE
adding service links + urls for JBA-v1-18

### PR DESCRIPTION
**Pull request by lzehl**
(+ small adjustment for AMBA)

### CODE REVIEW

**Comment by lzehl** [comment]
@xgui3783 I've added you to the SANDS development team since I expect you will contribute quite often in future. It also makes you a potential reviewer for PRs which I will directly make use of :wink: Could you review the service Link PR since you've provided the respective URLs?

**Comment by lzehl** [comment]
random samples should be enough for URLs and serviceLinks (they are automatically created, if there is an error it should be present in all).

File: instances/atlas/URL/JBA-v1-18/JBA-v1-18_Colin27-1998_Area-1_PM-v8-4_left-jsonld.jsonld [hunk 1]
```diff
@@ -0,0 +1 @@
+{"@context": {"@vocab": "https://openminds.ebrains.eu/vocab/"}, "@id": "https://openminds.ebrains.eu/instances/URL/JBA-v1.18_Colin27-1998_Area-1_PM-v8.4_left", "@type": "https://openminds.ebrains.eu/core/URL", "URL": "https://atlases.ebrains.eu/viewer/#/a:juelich:iav:atlas:v1.0.0:1/t:minds:core:referencespace:v1.0.0:7f39f7be-445b-47c0-9791-e971c0b6d992/p:minds:core:parcellationatlas:v1.0.0:94c1125b-b87e-45e4-901c-00daee7f2579/r:jubrain%20colin%20v18%20left::1z/@:0.0.0.-W000.._eCwg.2-FUe3._-s_W.2_evlu..7LIx..2pi7c~.1hyh1~.3SUsT..1LSm"}
```
Comment (xgui3783) [inline instances/atlas/URL/JBA-v1-18/JBA-v1-18_Colin27-1998_Area-1_PM-v8-4_left-jsonld.jsonld]: should we maybe have indented jsonld here (and other service link instances?)

Comment (lzehl) [reply instances/atlas/URL/JBA-v1-18/JBA-v1-18_Colin27-1998_Area-1_PM-v8-4_left-jsonld.jsonld]: yes... I don't know why it wasn't indented... I will fix that. 
What do you mean with "and other service link instances"?

**Comment by xgui3783** [comment]
Sorry, I meant other unidented files in this PR

File: instances/atlas/URL/JBA-v1-18/JBA-v1-18_Colin27-1998_Area-1_PM-v8-4_left-jsonld.jsonld (hunk 1 above)
Comment (lzehl) [reply instances/atlas/URL/JBA-v1-18/JBA-v1-18_Colin27-1998_Area-1_PM-v8-4_left-jsonld.jsonld]: @xgui3783 indent should be fixed

**Comment by lzehl** [comment]
@xgui3783 indent should be fixed for all now

**Review by xgui3783 — APPROVED** [review]
Looks good to me.

Am I correct in assuming that even though serviceLink is technically in openminds_core, it makes more sense to include the specific instances in sands repository? 

(I should also have siibraexplorer registered as an instance of service sooner rather than later oO)

**Comment by lzehl** [comment]
@xgui3783 yes. the plan is to move the instances soon to the main github, so that these issues (where is the schema to the instance) do not occur anymore. I might even do this change before xmas, but it depends on the workload...

**Comment by lzehl** [comment]
If you don't see any errors. Feel free to merge ;)

**Comment by xgui3783** [comment]
I have (not all of them, but selected few) and they seem to be working fine.

I believe I approved the PR?

**Comment by xgui3783** [comment]
Aww you already did. Perfect!

**Comment by lzehl** [comment]
:wink: (if you approve feel free to also merge directly)

**Pull request merged**